staging/src/k8s.io/apiextensions-apiserver/hack/update-codegen.sh

```diff
@@ -27,7 +27,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 CLIENTSET_NAME_VERSIONED=clientset \
 CLIENTSET_NAME_INTERNAL=internalclientset \
-"${CODEGEN_PKG}/generate-internal-groups.sh" all \
+"${CODEGEN_PKG}/generate-internal-groups.sh" deepcopy,client,lister,informer,conversion \
```

Comment:
why remove this?

Reply:
We add openapi to "all", but are not ready for apiextensions-apiserver yet. I have a follow-up to readd it.

Reply:
probably needs to be in the release notes for other consumers of code-gen?

Reply:
added